dags/solutions_team/configs/pax/solutionsteam_pax_supported_config.py

```diff
@@ -77,9 +77,7 @@ def get_runtime_version(pax_version: PaxVersion, tpu_version: TpuVersion) -> str
   elif tpu_version is TpuVersion.V5P:
     return RuntimeVersion.V2_ALPHA_TPUV5.value
   else:
-    if pax_version is PaxVersion.STABLE:
-      return RuntimeVersion.TPU_VM_V4_BASE.value
-    elif pax_version is PaxVersion.NIGHTLY:
+    if pax_version in [PaxVersion.STABLE, PaxVersion.NIGHTLY]:
```

Comment:
Do you think it's reasonable to set `RuntimeVersion.TPU_UBUNTU2204_BASE.value` as default for rest of TPU versions? As it works for both stable and nightly